modules/ssh-agent-using.adoc

```diff
@@ -232,13 +231,13 @@ $ eval "$(ssh-agent -s)"
 ----
 Agent pid 31874
 ----
-+
-ifndef::ibm-power-vs[]
-[NOTE]
-====
-If your cluster is in FIPS mode, only use FIPS-compliant algorithms to generate the SSH key. The key must be either RSA or ECDSA.
-====
-endif::ibm-power-vs[]
+//+
+//ifndef::ibm-power-vs[]
+//[NOTE]
+//====
+//If your cluster is in FIPS mode, only use FIPS-compliant algorithms to generate the SSH key. The key must be //either RSA or ECDSA.
```

Comment:
//either

Reply:
Probably bad wrapping on my side. I will take care of it.